modules/editor/message/MessageEditor.tsx

```diff
@@ -19,8 +22,21 @@ const DataEditorModal = dynamic<DataEditorModalProps>(async () =>
   import("../data/DataEditorModal").then(module => module.DataEditorModal),
 )
 
-const ErrorWrapper = styled.div`
-  margin: 8px 0 0;
+const Message = styled(Markdown)`
+  margin-top: -8px;
+  font-size: 15px;
+`
+
+const BottomActions = styled.div`
+  display: flex;
+  flex-flow: wrap;
+
+  margin-bottom: -8px;
+
+  & > * {
+    margin-right: 12px;
+    margin-bottom: 8px;
+  }
 `
```

Comment:
I notice this is copy pasted from the `Editor` component, can you instead move it to `common/layout/ButtonList.tsx`?